pkg/server/credvalidator/validator.go

```diff
@@ -166,20 +167,10 @@ func checkX509CertificateExpiration(cert *x509.Certificate, now time.Time) error
 }
 
 func hasExtKeyUsage(extKeyUsage []x509.ExtKeyUsage, want x509.ExtKeyUsage) bool {
```

Comment:
nit: we could probably just remove these functions now and inline the `slices.Contains` invocation.